schema/bom-1.6.schema.json

```diff
@@ -689,6 +689,7 @@
         },
         "licenses": {
           "title": "BOM License(s)",
+          "description": "The license information for the BOM document.\nThis may be different from the license(s) of the component that the BOM describes.",
```

Comment:
what do you mean?

Reply:
Sorry, let me try to clarify.

Maybe I'm wrong, but... a BOM can describe multiple components (e.g. a parent composed of multiple subcomponents), right?

It's a nit-pick and it might be wrong. I'm happy to just ignore my comment.

Reply:
> Maybe I'm wrong, but... a BOM can describe multiple components (e.g. a parent composed of multiple subcomponents), right?


Yes. But that license here (`$.metadata.licenses`) is not for any component. The license here is for the CycloneDX document itself. 

Use case: you produce a BOM and you do not want anybody outside your company to publish it — you'd add the appropriate license in this field, and you are good to go.


in contrast, the license of a component is on the `component` level -- https://cyclonedx.org/docs/1.5/json/#metadata_component_licenses

Reply:
Yes, I understand :)
Buutttt this is what you're saying:

> The license information for the BOM document.\nThis may be different from the license(s) of the component that the BOM describes.

And I wanted to suggest it could be:

> The license information for the BOM document.\nThis may be different from the license(s) of the component(s) that the BOM describes.

Reply:
ah, i see what you mean. 

```patch
- "description": "The license information for the BOM document.\nThis may be different from the license(s) of the component that the BOM describes.",
- "description": "The license information for the BOM document.\nThis may be different from the license(s) of the component(s) that the BOM describes.",
```

Reply:
This is not important really. I'll leave it up to you whether we want to change it :)

Reply:
done via 8a168a47a5533da80c6300c1be321ff991efa406